Book label

Kccendana opened the pull request:
- Create a Book class in a separate .rb file.
- Create a Label class with an association to the Item class (in a separate .rb file).
- All Book class properties visible in the diagram should be defined and set up in the constructor method.
- All Label class properties visible in the diagram should be defined and set up in the constructor method.

Kccendana committed:
Create add_item_info.rb

Kccendana committed:
Create book.rb

Kccendana committed:
Create books.json

Kccendana committed:
Update catalog_management.rb

Kccendana committed:
Update genres.rb

Kccendana committed:
Update inc_helper.rb

Kccendana committed:
Update item_class.rb

Kccendana committed:
Update main.rb

Kccendana committed:
Create label.rb

Kccendana committed:
Update menu.rb

Kccendana committed:
Update music_album.rb

Kccendana committed:
Create music_albums.json

Kccendana committed:
Update genre_rspec.rb

Kccendana committed:
Update item_rspec.rb

Commit:
Fix loading for the book.json

Kccendana committed:
modify

Kccendana committed:
modify books and label

Kccendana committed:
Update book.rb

Kccendana committed:
Update add_item_info.rb

Kccendana committed:
Update books.json

Kccendana committed:
Update catalog_management.rb

Kccendana committed:
Update inc_helper.rb

Kccendana committed:
Update item_class.rb

Kccendana committed:
Update label.rb

Kccendana committed:
Create labels.json

Kccendana committed:
added labels and book to the menu

Kccendana committed:
add tests for book label

Kccendana committed:
Merge branch 'develop' into book-label

Kccendana committed:
fix linter error

Kccendana committed:
fix linter error

Kccendana committed:
modify database

zbahati reviewed (approved):
# Status: Approved :green_circle: :1st_place_medal: :100:
![success](https://camo.githubusercontent.com/6ada59f6189497c1c2cae6d06f8154b7307a58643f232340e288140ed469eb28/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966)

### Hey @Kccendana 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Following Gitflow
✔️ Following Requirements

zbahati committed:
Merge branch 'develop' into book-label

Commit:
Fix rubocop errors

Commit:
Merge branch 'book-label' of https://github.com/Kccendana/catalog_of_my_things into book-labe

Commit:
ft book-label

Fixing merging conflict
Fixing linters errors
Adding the Module to handle rubocop erros
Fix Bugs when combining all of the files when submiting the book-label
